Tools/CISupport/ews-build/steps.py

```diff
@@ -2150,6 +2174,188 @@ def hideStepIf(self, results, step):
         return not self.doStepIf(step)
 
 
+class FilterPullRequestsIntoQueues(buildstep.BuildStep, GitHubMixin, AddToLogMixin):
+    name = 'filter-pull-requests-into-queues'
+
+    @defer.inlineCallbacks
+    def run(self):
+        num_prs = yield self.getNumPRs()
+        if num_prs:
+            sorted_prs = yield self.sortPRs(num_prs)
+            if sorted_prs:
+                list_of_prs, label_as_safe, label_as_blocked = sorted_prs
+            else:
+                return FAILURE
+            self.setProperty('safe_pr_list', label_as_safe)
+            yield self._addToLog('stdio', 'PRs to merge: {}.\n'.format(label_as_safe))
+            self.setProperty('blocked_pr_list', label_as_blocked)
+            yield self._addToLog('stdio', 'PRs to block: {}\n'.format(label_as_blocked))
+
+            if len(label_as_safe):
+                self.build.addStepsAfterCurrentStep([SafeMergeToMergeQueue()])
+            if len(label_as_blocked):
+                self.build.addStepsAfterCurrentStep([SafeMergeToMergeBlocked()])
+
+        defer.returnValue(SUCCESS if num_prs else FAILURE)
+
+    def getResultSummary(self):
+        if self.results == SUCCESS:
+            return {'step': f"Successfully requested and filtered pull requests"}
+        elif self.results == FAILURE:
+            return {'step': f"Failed to request and filter pull requests"}
+        return buildstep.BuildStep.getResultSummary(self)
+
+    @defer.inlineCallbacks
+    def getNumPRs(self, label='\"safe-merge-queue\"'):
+        query_body = '{ repository(owner:"Webkit", name:"WebKit") { pullRequests(labels: %s) { totalCount } } }' % label
+        query = {'query': query_body}
+        rc = yield self.query_graph_ql(query)
+        if rc:
+            num_prs = rc['data']['repository']['pullRequests']['totalCount']
+            yield self._addToLog('stdio', 'There are {} PR(s) in safe-merge-queue.\n'.format(num_prs))
+            defer.returnValue(num_prs)
+        else:
+            yield self._addToLog('stdio', 'Failed to retrieve number of PRs.\n')
+            defer.returnValue(None)
+
+    @defer.inlineCallbacks
+    def sortPRs(self, num_prs, label='\"safe-merge-queue\"'):
```

Comment:
what is this method doing? 
Doesn't look like it is doing exactly what the name says, which is sorting PRs. If it's getting list a of PRs, then the name should indicate so, e.g.: `getListOfPRsWithLabel` or `getSortedListOfPRsWithLabel` etc.